Export Analysis Reports/Sdl.Community.ExportAnalysisReports/ReportExporterControl.cs

```diff
@@ -577,45 +577,49 @@ private void GenerateReport()
 		{
 			try
 			{
-				if (!IsNullOrEmpty(reportOutputPath.Text) && Directory.Exists(reportOutputPath.Text))
+				if (!IsNullOrEmpty(reportOutputPath.Text))
 				{
+					Help.CreateDirectory(reportOutputPath.Text);
 					var projectsToBeExported = _projectsDataSource.Where(p => p.ShouldBeExported).ToList();
-					foreach (var project in projectsToBeExported)
+					var areCheckedLanguages = projectsToBeExported.Any(p => p.LanguagesForPoject.Any(l => l.Value));
```

Comment:
Here we can take the languages and use them bellow.

Reply:
The languages cannot be retrieved outside of foreach because it depends on the iterated project